_sub/security/cloudtrail-config/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "cloud_watch_logs_group_name" {
```

Comment:
CloudWatch is a name, so I think all variables ought to be prefixed with `cloudwatch_` instead of `cloud_watch_`